VAMobile/src/components/FormWrapper/FormFields/VATextInput.test.tsx

```diff
@@ -1,14 +1,14 @@
 import 'react-native'
 import React from 'react'
-import { Pressable, TextInput } from 'react-native'
+import { TextInput } from 'react-native'
 // Note: test renderer must be required after react-native.
 import 'jest-styled-components'
-import { ReactTestInstance, act } from 'react-test-renderer'
+import { ReactTestInstance } from 'react-test-renderer'
+import { screen } from '@testing-library/react-native'
 import Mock = jest.Mock
 
 import { context, render, RenderAPI, waitFor } from 'testUtils'
```

Comment:
All the `waitFor()`s can be removed from all the tests

Reply:
Updated.